ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -80,6 +80,34 @@ def test_tensorflow_eigh(
     )
 
 
+@handle_frontend_test(
+    fn_tree="tensorflow.linalg.eigvals",
+    dtype_and_input=_get_dtype_and_matrix(),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_eigvals(
+    *,
+    dtype_and_input,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x = dtype_and_input
+    assume(matrix_is_stable(x[0]))
+    if x[0].dtype == ivy.float32:
+        x[0] = x[0].astype("float64")
+        input_dtype = [ivy.float64]
+    helpers.test_frontend_function(
```

Comment:
Just read through your discord discussion. I'm pretty sure you can just go ahead and write an alternative check by assigning variables to the output of `test_frontend_function` and doing your own comparison.

Reply:
I've now written an alternative check and carried out my own comparison, where I sort the eigenvalues before comparing them. The tests were running quickly before when I was using Ivy's branch yesterday, but they are slower when I pulled the newest changes from Ivy.

Reply:
As you can see the tests are fast here^, working ontop of a of Ivy's master branch from the 13th of July